google-cloud-bigtable/lib/google/cloud/bigtable/instance_admin_client.rb

```diff
@@ -0,0 +1,424 @@
+# frozen_string_literal: true
+
+require "google/cloud/bigtable/admin/v2"
+require "google/bigtable/admin/v2/instance_pb"
+
+module Google
+  module Cloud
+    module Bigtable
+      # Protobuf aliases
+      Instance = Google::Bigtable::Admin::V2::Instance
+      Cluster = Google::Bigtable::Admin::V2::Cluster
+
+      class InstanceAdminClient
+        # Bigtable instance admin client for creating, configuring, and deleting
+        # instances and clusters
+
+        attr_reader :project_id, :options
+
+        # @param project_id [String]
+        # @param options [Hash]
+        def initialize project_id, options = {}
+          @project_id = project_id
+          @options = options
+        end
+
+        # Get instance information
+        #
+        # @param instance_id [String]
+        # @return [Google::Bigtable::Admin::V2::Instance]
+        # @example
+        #   require "google/cloud/bigtable"
```

Comment:
You are requiring `"google/cloud/bigtable"`, which is what we expect to see. But I don't see that file in this PR yet.